pypi/LICENSE

```diff
@@ -0,0 +1,27 @@
+Boolector is a Satisfiability Modulo Theories (SMT) Solver for bit-vectors,
```

Comment:
Can we use https://github.com/Boolector/boolector/blob/master/COPYING instead of having a separate copy of the license?

Reply:
Good point. I'll update the build script to grab COPYING from the root of the tree